compiler/next/include/chpl/AST/FnCallExpr.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_AST_FNCALLEXPR_H
+#define CHPL_AST_FNCALLEXPR_H
+
+#include "chpl/AST/CallExpr.h"
+
+#include <vector>
+
+namespace chpl {
+namespace ast {
+
+/**
+  This class represents a function call or method call, including virtual and
+  non-virtual calls (since the difference is not known before resolution).
+ */
```

Comment:
As I mentioned above, this class is restricted to those calls that use parens, so I suggested using a different name for it to reflect parens.

Reply:
Did something in the code here indicate that it would be restricted to calls using parens? Or are you saying you would prefer that?

Reply:
I responded under ASTClassesList.h above: https://github.com/chapel-lang/chapel/pull/17587/files#r622270614